packages/ws-worker/src/util/log-final-reason.ts

```diff
@@ -12,7 +12,7 @@ export default async (context: Context, reason: ExitReason) => {
 
   await onJobLog(context, {
     time,
-    message: [message],
+    message: JSON.stringify([message]),
```

Comment:
The construction of log messages is really fiddly now.

Options:
1) Create a nice helper called `sendLog(level, message, name?)`, which handles the bigint and the string parsing
2) Be more lenient about the shape of message. I mean, it doesn't HAVE to be JSON - if it's a string we parse it as JSON, if its an array we leave it as JSON

I actually think we want a little of both...